src/managers/gatewayCache.ts

```diff
@@ -14,14 +14,13 @@ export class GatewayCache {
     if (cacheName !== undefined) this.cacheName = cacheName
   }
 
-  async get(key: string): Promise<undefined | any> {
+  async get<T>(key: string): Promise<undefined | T> {
     const result = await this.client.cache.get(this.cacheName, key)
```

Comment:
in the code you allow `cache.get` to have a type parameter `T`
maybe change it to `await this.client.cache.get<T>(this.cacheName, key)`?

Reply:
wdym?

Reply:
like wouldn't it make more sense to do `return await this.client.cache.get<T>(this.cacheName, key)` rather
`return result as T`

Reply:
Right, I'll do that